Client/game_sa/CCameraSA.cpp

```diff
@@ -446,8 +446,14 @@ float CCameraSA::GetShakeForce()
 
 void CCameraSA::ShakeCamera(float radius, float x, float y, float z) noexcept
 {
-    DWORD               dwFunc = FUNC_ShakeCam;
-    CCameraSAInterface* cameraInterface = GetInterface();
+    static CCameraSAInterface* cameraInterface = GetInterface();
+    if (radius <= 0.0f)
+    {
+        cameraInterface->m_fCamShakeForce = 0.0f;
```

Comment:
```suggestion
        GetInterface()->m_fCamShakeForce = 0.0f;
```

Reply:
I pass `cameraInterface` later in the inline assembly

Reply:
This is less readable and conflicts with current SA definitions.
Its better to leave it as inline assembly for now

Reply:
We are trying to avoid inline assembly for simple function calls.

- C++ is more readable and maintainable than assembly
- It's much easier to write c++ definition than assembly for developers
- It generates better code. Compiler cannot optimize functions with inline assembly
- You can avoid hacks with static variables

Reply:
Technically, You should make `resetCameraShake` function to be consistent with current API.
+ You should reset this effect when a player join a server

Reply:
sure